test/gtest/uct/test_ib.cc

```diff
@@ -442,7 +442,7 @@ UCS_TEST_P(test_uct_wakeup_ib, tx_cq)
     send_msg_e1_e2();
 
     /* make sure the file descriptor is signaled once */
-    ASSERT_EQ(poll(&wakeup_fd, 1, 10000), 1);
+    ASSERT_EQ(poll(&wakeup_fd, 1, 1000*ucs::test_time_multiplier()), 1);
```

Comment:
I don't think so. I believe that's OK if events are triggered after some time. This is particularly noticeable under valgrind.